papaparse.js

```diff
@@ -941,7 +952,19 @@ License: MIT
 			// when too many items have been added without their
 			// callback being invoked
 			parseCallbackQueue.push(bindFunction(function() {
-				this.parseChunk(typeof chunk === 'string' ? chunk : chunk.toString(config.encoding));
+				var encoding = config.encoding;
```

Comment:
I see this code is an exact copy of the other version. 
It's not better to encapsulate it on a function?